enos/modules/install_binaries/variables.tf

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+variable "artifactory_username" {
+  type        = string
+  description = "The username to use when connecting to artifactory"
+  default     = null
+}
+
+variable "artifactory_token" {
+  type        = string
+  description = "The token to use when connecting to artifactory"
+  default     = null
+  sensitive   = true
+}
+
+variable "artifactory_host" {
+  type        = string
+  description = "The artifactory host to search for Nomad artifacts"
+  default     = "https://artifactory.hashicorp.engineering/artifactory"
+}
+
+variable "artifactory_repo" {
+  type        = string
+  description = "The artifactory repo to search for Nomad artifacts"
+  default     = "hashicorp-crt-staging-local*"
+}
+
+variable "edition" {
+  type        = string
+  description = "The edition of the binary to search (one of ce or ent)"
+
+  validation {
+    condition     = contains(["ent", "ce"], var.edition)
+    error_message = "must be one of ent or ce"
+  }
+}
+
+variable "oss" {
+  type        = list(string)
```

Comment:
Can this by a `set(string)` so that if both the server and the matrix are Linux, we only need to download once?